scripts/zones/Beadeaux/IDs.lua

```diff
@@ -56,6 +59,13 @@ zones[xi.zone.BEADEAUX] =
         {
             [17379625] = 17379626, -- 139.642 -2.445 161.557
         },
+        FOR_THE_BIRDS_MOBS =
+        {
+            17379784,
+            17379785,
+            17379786,
+            17379787,
+        }
```

Comment:
This PR is going to need a rebase once this PR (#5736) gets merged.

This block of logic will also have to be re-done to not use raw IDs (see that PR to see an example on how to do this)

Reply:
Sounds good. 👍🏼

As far as using GetFirstID/not using raw IDs, it looks like these mobs (Nickel and Magnus Quadav) don't have unique names, do you have an example in mind for how to implement in this case?

Reply:
`FOR_THE_BIRDS_MOBS = GetFirstID('Magnes_Quadav') -- For The Birds Quest`

Then inside the quest lua you can do
```
local mobListTable =
{
    id.mob.FOR_THE_BIRDS_MOBS,
    id.mob.FOR_THE_BIRDS_MOBS + 1,
    id.mob.FOR_THE_BIRDS_MOBS + 2,
    id.mob.FOR_THE_BIRDS_MOBS + 3,
}
```
Then just replace all your `FOR_THE_BIRDS_MOBS` with `mobListTable` or whatever you think the table should be called

Reply:
Hmm, okay so I think I just misunderstood what GetFirstID was doing, this is limited to the zone for which the ID.lua file is for which would prevent conflicts with, for example, Magnes_Quadav in Beadeaux [s]?

Reply:
Correct, this only pulls from the same zone, but does not discriminate between mobs and NPCs